api/src/test/java/io/opentelemetry/api/baggage/BaggageTest.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.api.baggage;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.opentelemetry.context.Context;
+import org.junit.jupiter.api.Test;
+
+class BaggageTest {
+  @Test
+  void current_empty() {
+    Context.root().makeCurrent();
```

Comment:
try/resources the scopes - guessing you had to add the root.makeCurrent to the other tests because of missing closing here but we should have tests not interfere with each other :)

Reply:
Though I don't think I'd make the root current here anyways, can leave this out to test "the default".

Reply:
> Though I don't think I'd make the root current here anyways, can leave this out to test "the default".

assuming everyone else has properly closed their scopes! unlike me, apparently.